version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "2.16.0"
+version in ThisBuild := "2.16.1-SNAPSHOT"
```

Comment:
Is this supposed to say `SNAPSHOT` or should is just by `2.16.1`?

Reply:
snapshot now and remove snapshot in two steps is proper.  We skip a lot these days.